packages/sdk/src/modules/hosting/hosting-website.ts

```diff
@@ -0,0 +1,146 @@
+import { AxiosInstance } from 'axios';
+import { listFilesRecursive, uploadFilesToS3 } from '../../lib/common';
+import { DeployToEnvironment } from '../../types/hosting';
+
+export class HostingWebsite {
```

Comment:
We should rename this to just `Website` Or we should rename `File` to `StorageFile` etc.